helm-chart-sources/pulsar/templates/broker-sts/broker-sts-configmap.yaml

```diff
@@ -222,6 +222,10 @@ data:
   gcsManagedLedgerOffloadReadBufferSizeInBytes: "{{ .Values.storageOffload.readBufferSizeInBytes }}"
   {{- end }}
 {{- end }}
+{{- end }}
+  # Transactions
+{{- if .Values.brokerSts.transactionCoordinator.enabled }}
+  PULSAR_PREFIX_transactionCoordinatorEnabled: "true"
```

Comment:
Do we need to enable system topics to make this work out of the box on 2.10?

Reply:
yes
the idea is to just have a single flag to make transactions work on the cluster

Reply:
@michaeljmarshall is saying that you have to enable system topics as well.

but I can't remember this requirement for the transaction coordinator

Reply:
oh you're right. 

>but I can't remember this requirement for the transaction coordinator

me too, also checking in the code I don't see any strict relation